Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,132 @@ Meaning of columns:
+| `IfcLightSourceSpot`                  | :x:                  | -                          | -                     | -     |
+| `IfcLine`                             | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcLineSegment2D`                    | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcLoop`                             | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcMappedItem`                       | :heavy_check_mark:   | `RepresentationConverter`  | :heavy_check_mark:    | -     |
+| `IfcOffsetCurve2D`                    | :x:                  | -                          | -                     | -     |
+| `IfcOffsetCurve3D`                    | :x:                  | -                          | -                     | -     |
+| `IfcOffsetCurveByDistances`           | :x:                  | -                          | -                     | -     |
+| `IfcOpenShell`                        | :x:                  | -                          | -                     | -     |
+| `IfcOrientationExpression`            | :heavy_check_mark:   | `PlacementConverter`       | :x:                   | -     |
+| `IfcOrientedEdge`                     | :x:                  | -                          | -                     | -     |
+| `IfcOuterBoundaryCurve`               | :x:                  | -                          | -                     | -     |
+| `IfcPath`                             | :x:                  | `RepresentationConverter`  | -                     | -     |
+| `IfcPcurve`                           | :heavy_check_mark:   | `CurveConverter`           | :x:                   | -     |
+| `IfcPlanarExtent`                     | :x:                  | -                          | -                     | -     |
+| `IfcPlanarBox`                        | :x:                  | -                          | -                     | -     |
+| `IfcPlane`                            | :heavy_check_mark:   | `FaceConverter`            | :x:                   | -     |
+| `IfcPointByDistanceExpression`        | :heavy_check_mark:   | `PlacementConverter`       | :x:                   | -     |
+| `IfcPointOnCurve`                     | :heavy_check_mark:   | `PlacementConverter`       | :x:                   | -     |
+| `IfcPointOnSurface`                   | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcPolyline`                         | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcPolyLoop`                         | :heavy_check_mark:   | `CurveConverter`           | :heavy_check_mark:    | -     |
+| `IfcPolygonalBoundedHalfSpace`        | :x:                  | `CurveConverter`           | -                     | -     |
```

Comment:
I doubt that this is in the CurveConverter.

Reply:
Resolved 3972ec7